webapp-django/crashstats/signature/views.py

```diff
@@ -43,6 +43,43 @@
 )
 
 
+def get_date_range(params):
+    now = datetime.datetime.utcnow().replace(tzinfo=utc)
+
+    if not params.get('date'):
+        return (now - datetime.timedelta(days=7), now)
```

Comment:
Perhas instead of `7` we can make it `settings.DEFAULT_SIGNATURE_REPORT_RANGE_DAYS`?